lib/jazzy/config.rb

```diff
@@ -84,6 +84,11 @@ class << self
                     'delete the ~/Desktop directory.'],
       default: false
 
+    config_attr :objc_mode,
+      command_line: ['--[no-]objc'],
+      description: ['Generate docs for Objective-C.'],
```

Comment:
Braces are optional for single-item `command_line` and `description`. Not sure if it matters.